conans/client/cache/cache.py

```diff
@@ -93,19 +93,8 @@ def base_folder(self, ref):
         """
         return self.package_layout(ref).base_folder()
 
-    def conanfile(self, ref):
-        return self.package_layout(ref).conanfile()
-
-    def builds(self, ref):
-        return self.package_layout(ref).builds()
-
-    def build(self, pref, short_paths=False):
-        return self.package_layout(pref.ref, short_paths).build(pref)
-
-    def packages(self, ref):
-        return self.package_layout(ref).packages()
-
     def package(self, pref, short_paths=False):
+        # TODO: This is deprecated, only used in testing
```

Comment:
Add a `@deprecation.deprecated(deprecated_in="1.15", removed_in="xxx", details="...")`, or open an issue, assign it to me and I will deal with it for the future release... 😊